pom.xml

```
@@ -73,7 +73,7 @@
         <bukkitplugin.authors>Xephi, sgdc3, DNx5, timvisee, games647, ljacqu, Gnat008</bukkitplugin.authors>
 
         <!-- Change Bukkit Version HERE! -->
-        <bukkit.version>1.13-R0.1-SNAPSHOT</bukkit.version>
+        <bukkit.version>1.13.1-R0.1-SNAPSHOT</bukkit.version>
```

```suggestion
        <bukkit.version>1.13.2-R0.1-SNAPSHOT</bukkit.version>
```
Perhaps catch up the version before merging?